teamname-teamname-teamname/BangazonCLI
# added test for activeCustomer

## Status
**READY**

## Migrations
 NO

#### Moved test to customer manager on second commit. Wrapped it in a method on the third. Minor refactoring on a forth. Refactored tests to accomodated for the static classes ReturnActiveCustomer() and SelectActiveCustomer().   Deleted Banagazon_ActiveCustomerShould.cs. Code is properly commented.

## Description
Added public static class for Active Customer to the Customer Manager and set up a test (Bangazon_ActiveCustomerShould.cs) to return an int for all active customer. This will be refactored to account for our data.   

## Related PRs
List related PRs against other branches:

branch | PR


## Resolves Issue Number
number https://github.com/teamname-teamname-teamname/BangazonCLI/issues/2


## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

```sh
git checkout master
git fetch --all
git checkout jb-selectActiveCustomer

After pulling down my branch to test, make sure to have a bangazoncli_test.db file. 
Make sure you are in test/Bangazon.Tests and run dotnet test
```

1.

## Impacted Areas in Application
List general components of the application that this PR will affect:

src/Managers/CustomerManager.cs
test/Bangazon.Tests/Bangazon_CustomerManagerShould.cs
*
